bin/check-puppet-last-run.rb

```diff
@@ -95,7 +107,11 @@ def run
       @message += " with #{@failures} failures"
     end
 
-    if @now - @last_run > config[:crit_age] || @failures > 0
+    if @restart_failures > 0
```

Comment:
If you look at line 91 it always defaults to 0 when option to report restarts is not enabled. Let me know if you want this to be still changed.